package-lock.json

```diff
@@ -0,0 +1,10000 @@
+{
```

Comment:
Can we remove the `package-lock.json` please?  I've nothing against npm but ts-loader uses `yarn.lock` and I want to avoid maintaining 2 separate lock files.  Thanks!